drivers/classic/src/initialize_atmos.c

```diff
@@ -466,15 +419,18 @@ initialize_atmos(atmos_data_struct    *atmos,
                 }
             }
             else {
-                /* Local sub-daily forcings will be hourly for coding convenience
-                   Sub-daily forcings need to a) start at hour 0, local time
+                /* Local sub-daily forcings will be sub-daily.
+                   Sub-daily forcings need to a) start at sec 0, local time
                    and b) draw from the correct element of the supplied
                    forcings (if the supplied forcings are not in local time) */
-                fstepspday = HOURS_PER_DAY /
-                             param_set.FORCE_DT[param_set.TYPE[type].SUPPLIED -
-                                                1];
-                for (uidx = 0; uidx < (Ndays_local * HOURS_PER_DAY); uidx++) {
-                    k = (uidx - global_param.starthour + hour_offset_int) /
+                fstepspday =
+                    param_set.force_steps_per_day[param_set.TYPE[type].SUPPLIED
+                                                  -
+                                                  1];
+                for (uidx = 0;
+                     uidx < (Ndays_local * atmos_steps_per_day);
+                     uidx++) {
+                    k = (uidx - global_param.startsec + sec_offset_int) /
```

Comment:
OK - let me see if there is a way to preferentially NOT break within `[...]', cause we'll run into this everytime. As said, the other option is to get it the way we want and then discard whitespace only changes in git (need to check as well whether that is possible).